packages/react/src/tooltip/root/TooltipRoot.test.tsx

```diff
@@ -421,90 +421,6 @@ describe('<Tooltip.Root />', () => {
     });
   });
 
-  describe.skipIf(isJSDOM)('prop: onOpenChangeComplete', () => {
```

Comment:
I ran into this as I was debugging the scheduler, these tests seem to be a duplicate (with one block of code less inside one of the `it()`, and a few less `it()` blocks) of the tests just below. I've kept the longer version of these below.